cloud-controller-manager/do/cloud.go

```diff
@@ -147,8 +156,23 @@ func (c *cloud) Initialize(clientBuilder cloudprovider.ControllerClientBuilder,
 
 	sharedInformer.Start(nil)
 	sharedInformer.WaitForCacheSync(nil)
+
 	go res.Run(stop)
 	go c.serveDebug(stop)
+
+	if c.resources.publicAccessFirewallName == "" {
+		klog.Infof("Nothing to manage since firewall name was not provided")
```

Comment:
Should be `klog.Info` since we do not have any placeholders.

Reply:
Nice catch. I've also removed a few others in the `firewall_controller_test.go` file.